lib/iex/lib/iex/helpers.ex

```diff
@@ -22,30 +22,31 @@ defmodule IEx.Helpers do
-    * `flush/0`       - flushes all messages sent to the shell
-    * `h/0`           - prints this help message
-    * `h/1`           - prints help for the given module, function or macro
-    * `i/1`           - prints information about the given data type
-    * `import_file/1` - evaluates the given file in the shell's context
-    * `l/1`           - loads the given module's beam code
-    * `ls/0`          - lists the contents of the current directory
-    * `ls/1`          - lists the contents of the specified directory
-    * `nl/2`          - deploys local beam code to a list of nodes
-    * `pid/1`         - creates a PID from a string
-    * `pid/3`         - creates a PID with the 3 integer arguments passed
-    * `pwd/0`         - prints the current working directory
-    * `r/1`           - recompiles the given module's source file
-    * `recompile/0`   - recompiles the current project
-    * `respawn/0`     - respawns the current shell
-    * `s/1`           - prints spec information
-    * `t/1`           - prints type information
-    * `v/0`           - retrieves the last value from the history
-    * `v/1`           - retrieves the nth value from the history
+    * `b/1`                   - prints callbacks info and docs for a given module
+    * `c/1`                   - compiles a file into the current directory
+    * `c/2`                   - compiles a file to the given path
+    * `cd/1`                  - changes the current directory
+    * `clear/0`               - clears the screen
+    * `flush/0`               - flushes all messages sent to the shell
+    * `h/0`                   - prints this help message
+    * `h/1`                   - prints help for the given module, function or macro
+    * `i/1`                   - prints information about the given data type
+    * `import_file/1`         - evaluates the given file in the shell's context
+    * `import_if_available/2` - imports the given module only if available
+    * `l/1`                   - loads the given module's beam code
+    * `ls/0`                  - lists the contents of the current directory
+    * `ls/1`                  - lists the contents of the specified directory
+    * `nl/2`                  - deploys local beam code to a list of nodes
+    * `pid/1`                 - creates a PID from a string
+    * `pid/3`                 - creates a PID with the 3 integer arguments passed
+    * `pwd/0`                 - prints the current working directory
+    * `r/1`                   - recompiles the given module's source file
+    * `recompile/0`           - recompiles the current project
+    * `respawn/0`             - respawns the current shell
+    * `s/1`                   - prints spec information
+    * `t/1`                   - prints type information
+    * `v/0`                   - retrieves the last value from the history
+    * `v/1`                   - retrieves the nth value from the history
```

Comment:
I believe since we are changing all these lines to align them (thus affecting the git blame anyways), we should stop aligning them at all, e.g, use `* `helper` - ...` everywhere. Wdty? \cc @lexmag